src/components/navbar/navDropdown/NavDropdown.module.scss

```diff
@@ -0,0 +1,60 @@
+@import '../../../utils/variables';
+
+.dropdownContainer {
+    z-index: 1;
+    background-color: $white;
+    position: absolute;
+    display: inherit;
+    top: 0;
+    left: calc(5vw - 10px);
+    width: 90vw;
+    margin: 0 10px 10px 10px;
+    padding: 15px;
+    border-radius: 10px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    box-shadow: 0px 4px 6px 5px rgba(0, 0, 0, 0.1);
+
+    transition: all 0.5s ease-out; //inline style animation
+
+    @media (min-width: $medium) {
+        display: none;
+    }
+    @media (max-height: 500px) {
```

Comment:
Use the existing screen sizes please. `$small` is `480px` so should do the job.

Reply:
the app breaks at specific sizes. I don't feel comfortable using fixed sizes, In this scenario, someone with a max-height between 480px and 500px and a small width will experience a screen where they would have a really awkward experience.
I like the screen sizes because it lets me know what are target client sizes are, but I feel like too much of the app breaks between these. I prefer to make the site fully responsive and not responsive to specific sizes only.

Reply:
Hmm, that is an interesting point! Could we resolve this by having a $medium (this may already exist) size or something like that, or do we need to be more flexible than that?